### ISSUE TITLE
What is ajde.ts file?

### ISSUE THREAD

**Issue by gabrieldesousah** [JohnWeidner/ScreenRecorderExtension#16]
Hi, I am trying to understand this project to make some updates, and I am with some doubts... 
- the file ajde.ts does what? Is a file to be converted in ourrecorder.js? It's used?

**Comment by zombiestudio** [comment]
Yes, as I remember "ajde.ts" was source code for "ourrecorder.js"
You should do all updates to "ajde.ts" then compile it to 
"ourrecorder.js"
